src/ZODB/blob.py

```diff
@@ -942,12 +942,16 @@ def remove_committed_dir(path):
                 filename = os.path.join(dirpath, filename)
                 remove_committed(filename)
         shutil.rmtree(path)
-
     link_or_copy = shutil.copy
```

Comment:
One last niggle:  the empty line separates a function definition from the assignment, so please restore it.